js/components/conceptsetInclusionCount/conceptsetInclusionCount.js

```diff
@@ -23,6 +23,11 @@ define([
 
 			this.getInclusionCount = this.getInclusionCount.bind(this);
 			this.conceptSetExpression.subscribe(this.getInclusionCount);
+			this.conceptSetExpression().forEach((item) => {
+				item.includeDescendants.subscribe(this.getInclusionCount);
+				item.includeMapped.subscribe(this.getInclusionCount);
+				item.isExcluded.subscribe(this.getInclusionCount);
+			});
```

Comment:
@wivern , cannot we apply more universal `ko.computed(() => ko.toJSON(root)).subscribe(this.getInclusionCount)` instead of 6 lines above?